mvnw

```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```

Comment:
Why mvn wrapper is in need?

Reply:
Others can build jars based on this java code with maven without the need to build the maven environment, and shield them from build differences caused by inconsistent maven versions.